### PR TITLE
Update neural machine translation with transformer example for Keras 3.

### PR DESCRIPTION
**Pull request by hertschuh**

### CODE REVIEW

File: examples/nlp/neural_machine_translation_with_transformer.py [hunk 1]
```diff
@@ -32,15 +32,34 @@
...
+
+import tensorflow.data as tf_data
+import tensorflow.strings as tf_strings
+
+import keras
+from keras import layers
+from keras import ops
+from keras.layers import TextVectorization
+
+keras.config.disable_traceback_filtering()
```
Comment (fchollet) [inline examples/nlp/neural_machine_translation_with_transformer.py:62]: You can remove this file in all 3 files (manually, no need to regenerate)

Comment (hertschuh) [reply examples/nlp/neural_machine_translation_with_transformer.py:62]: Oops, this was me debugging. Will do.

Comment (hertschuh) [reply examples/nlp/neural_machine_translation_with_transformer.py]: Done

**Review by fchollet — APPROVED** [review]
LGTM

**Pull request merged**